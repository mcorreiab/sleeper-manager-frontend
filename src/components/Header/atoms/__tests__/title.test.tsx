import renderer from "react-test-renderer";
import Title from "../title";

it("should render the site title", () => {
  const tree = renderer.create(<Title />).toJSON();
  expect(tree).toMatchSnapshot();
});
