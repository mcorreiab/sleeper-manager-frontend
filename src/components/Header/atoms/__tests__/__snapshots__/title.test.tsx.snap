// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the site title 1`] = `
<h1
  className="text-sm-orange m-0 text-base"
>
  Sleeper Manager
</h1>
`;
